config/branch_protector/rules.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branch-protection:
```

Comment:
Which branches are protected, and how?  We protect `master` and `release-*` currently.

Reply:
The current config file will protect all the branches. But if we want to exclude specific branches, we can configure them with the `exclude` field like https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L139-L142.